src/game.ts

```diff
@@ -13,23 +14,49 @@ export interface Game {
   hasMatchAllCards(): boolean;
 }
 
+interface DifficultyLevel {
+  numCards: number,
+  matchLength: number,
+}
+
+export enum Difficulty {
+  EASY = 'easy',
+  MEDIUM = 'medium',
+  HARD = 'hard'
+}
+
+function getDifficultySpec(difficulty: Difficulty): DifficultyLevel {
+  switch (difficulty) {
+    case Difficulty.EASY:
+      return { numCards: 2, matchLength: 2 };
+    case Difficulty.MEDIUM:
+      return { numCards: 4, matchLength: 2 };
+    case Difficulty.HARD:
+      return { numCards: 8, matchLength: 2 };
+  }
+}
+
 interface GameData {
   start: Date | null;
   end: Date | null;
   score: number;
   attempts: number;
   cards: Card[];
+  difficulty: Difficulty;
 }
 
 const DefaultGameData = {
   start: null,
   end: null,
   score: 0,
   attempts: 0,
-  cards: getInitialCards()
-};
+  cards: getInitialCards(getDifficultySpec(Difficulty.EASY).numCards),
+  difficulty: Difficulty.EASY
```

Comment:
@HighPharoah Is this leftover from development? I’d vote for defaulting to more than 2 pairs here.

Reply:
good catch, updated